packages/@ngtools/webpack/src/type_checker_worker.ts

```diff
@@ -1,86 +1,70 @@
-// @ignoreDep typescript
-import * as process from 'process';
-import * as ts from 'typescript';
 import chalk from 'chalk';
-
-import { WebpackCompilerHost } from './compiler_host';
-import { time, timeEnd } from './benchmark';
-import { CancellationToken, gatherDiagnostics } from './gather_diagnostics';
+import * as ts from 'typescript';
+import * as process from 'process';
+import { WebpackCompilerHost } from '@ngtools/webpackcompiler_host';
+import { time, timeEnd } from '@ngtools/webpackbenchmark';
+import { CancellationToken, gatherDiagnostics } from '@ngtools/webpackgather_diagnostics';
```

Comment:
I think something went wrong while renaming these imports. They should still be relative.

Reply:
Thanks! Should be fixed in a moment.